CHANGELOG.rst

```diff
@@ -15,6 +15,7 @@ Version 2022-dev
 -  add missing cmath include (#351)
 -  removed eigenvalue algorithm, now done by eigen  (#352)
 -  Added N-DimVector (#360)
+-  properties can be removed and string conversion much improved (#365)
```

Comment:
Huh? Wrong number?

Reply:
Yes this is a part of the PR which will be merged later, because some XTP functionality does not allow this change atm

Reply:
Maybe add a line for #366 then as well.